9/DataTypes.md

```diff
@@ -326,3 +326,25 @@ Results of failed operations (exceptions) within the Tango Controls are sent as
  desc = *VCHAR ; SHOULD describe the failure in more details
  origin = *VCHAR ; SHALL identify the operation or the tango object which caused the failure, eg. function name
 ```
+
+* MultiDevFailed type name is as follows:
+```abnf
+ MultiDevFailed = "MultiDevFailed"
+```
+
+* Any value of MultiDevFailed type SHALL follow \<milti-dev-failed-value\> rule:
+```abnf
+ multi-dev-failed-value = 1*named-dev-error 
+
+ named-dev-error = name index-in-call 1*error
+
+ name = *VCHAR
+ index-in-call = num-val ; of long data type in range of -(2^31) to (2^31)-1
```

Comment:
Indeed, an index being a long as specified by the IDL. But I guess we will get some troubles if people start using negative indices?

Reply:
Should I change it to 0 - ...?

Reply:
Good question! I don't know what's the  best.
-(2^31) to (2^31)-1 is the corresponding range for the long type but I guess in this specific case, we won't use (and probably don't want to use) negative values?

Reply:
Then I suggest this:
```suggestion
 index-in-call = num-val ; of long data type (32 bits) in the range of 0 to (2^31)-1
```

Reply:
@piogor, fine with me.